### DISCUSSION TITLE
why a non-commercial license?

### DISCUSSION THREAD

**Discussion by olliacc** [PKU-YuanGroup/Open-Sora-Plan#26] [Q&A] [answered]
why a non-commercial license? I thought an open-source license should not discriminate against a legal use.

**Comment by ttttt12345** [comment]
可以講人話嗎 超可悲

**Comment by ttttt12345** [comment]
在狗叫沈ㄇㄜ˙

**Comment by wuhanstudio** [comment, marked as answer]
> This project aims to create a simple and scalable repo, to reproduce [Sora](https://openai.com/sora) (OpenAI, but we prefer to call it "CloseAI" )

毕竟人家在 README 里嘲笑 OpenAI 是 CloseAI 了：人家不开源，害得我们只能发动社区的力量复制一份。

等我们千辛万苦复制出来了，就是我们的自己的 Sora，当然要提前设定好 Non-Commercial license，不然又被自己人复制走了怎么办 T_T

Update：Thanks for your concern. 已经换成 MIT License 了。
